src/Logging.AzureAppServices/src/BlobLoggerProvider.cs

```diff
@@ -58,10 +58,20 @@ public BlobLoggerProvider(IOptionsMonitor<AzureBlobLoggerOptions> options)
         internal override async Task WriteMessagesAsync(IEnumerable<LogMessage> messages, CancellationToken cancellationToken)
         {
             var eventGroups = messages.GroupBy(GetBlobKey);
+            DateTime currDate = DateTime.Now;
             foreach (var eventGroup in eventGroups)
             {
                 var key = eventGroup.Key;
-                var blobName = $"{_appName}/{key.Year}/{key.Month:00}/{key.Day:00}/{key.Hour:00}/{_fileName}";
+                //var blobName = $"{_appName}/{key.Year}/{key.Month:00}/{key.Day:00}/{key.Hour:00}/{_fileName}";
```

Comment:
Why is this commented out?

Reply:
That code was replaced with the logic from line 66 - 74 because it forces nested folder in Azure blob. Line 71 produces a flat folder structure, while line 74 produces the original nested folder structure where the code was commented out.

Reply:
Shouldn't it be removed then? Why did you leave the code in but commented out?

Reply:
Yes you're right, it should be removed. I will remove it and commit my change

Reply:
@davidfowl : I removed the commented out code, committed and pushed. This is my first time contributing, and I was not sure whether to leave commented out original code or to delete. But now I know to go ahead and delete when deemed not needed. Thank you.

Reply:
@davidfowl : After removing the commented our code, it would not pass the checks. Could you please advise what might have gone wrong, could this be something on the CI side?

Reply:
@davidfowl : The build is ready for your review. Could you please revisit this PR? Thank you!